### DISCUSSION TITLE
Change "version" name?

### DISCUSSION THREAD

**Discussion by asherevan** [russhughes/st7789_mpy#173] [Q&A]
When I compile the firmware and open the REPL, it prints this:

`
MicroPython v1.24.0-preview.309.gf1bdac375.dirty on 2024-09-17; Generic ESP32 module with ESP32
`

Is there a way to customise what this says?

**Comment by walklinewang** [comment]
You can modify board name and mcu name in [this](https://github.com/micropython/micropython/blob/master/ports/esp32/boards/ESP32_GENERIC/mpconfigboard.h) file, otherwise, you can try to search `help()` in MicroPython's source code.

**Comment by asherevan** [comment]
I tried searching `help()`, but it couldn’t find anything.
